### ISSUE TITLE
VSCode Omnisharp tab formatting issue

### ISSUE THREAD

**Issue by tilkinsc** [OmniSharp/omnisharp-roslyn#2575]
So I enabled omnisharp (after having it disabled because I am heckin vanilla). I am not using the C# dev kit extension (idk who its targeted for, me?).

I got greeted by my code looking like this,
![image](https://github.com/OmniSharp/omnisharp-roslyn/assets/7494772/1dae1767-114f-428c-a7b2-09a5913f90b7)

I was like what the heck, so in looking for a reason it apparently hates me using tabs. So I am like no big deal tab issues theres options for that™️.
I went over to omnisharp.json in my %userprofile%/.omnisharp and set some settings
```json
	"FormattingOptions": {
		"EnableEditorConfigSupport": false,
		"OrganizeImports": true,
		"NewLine": "\n",
		"UseTabs": true,
		"TabSize": 4,
		"IndentationSize": 4,
```

However, I am not able to make anything happen. I can enable/disable inlay hints. Yet it ignores my use of tabs entirely. I've attempted to disable editor config support, enable it, in both places. Its actually pretty ambiguous what that means.

One final thing,
![image](https://github.com/OmniSharp/omnisharp-roslyn/assets/7494772/faee30c1-325b-4873-a464-a9b43ab72494)

I refuse to use block scoped namespaces.

I like some of the features omnisharp has to offer but this is unusable. Solutions?

**Comment by JoeRobich** [comment]
I recommend using an .editorconfig file for storing your formatting settings along with the [editorconfig vs code extension](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig). The Roslyn formatter, which is used by O#, will respect your .editorconfig. Do note that the Roslyn formatter trims trailing whitespace which may be what you are calling "tab pits". The C# extension has a long running tracking issue waiting for virtual space to be implemented in vscode (see https://github.com/dotnet/vscode-csharp/issues/1980).

**Comment by tilkinsc** [comment]
Yeah I stumbled upon the editorconfig which works very well. I know virtual space is a solution to the tab pits problem, but there is also the solution of "dont touch my freakin whitespace".

**Comment by JoeRobich** [comment]
> Yeah I stumbled upon the editorconfig which works very well. I know virtual space is a solution to the tab pits problem, but there is also the solution of "dont touch my freakin whitespace in the first place".

As we use the Roslyn formatter, this would be a good issue to comment on https://github.com/dotnet/roslyn/issues/41666
